### PR TITLE
feat: option to disable 

### PR DESCRIPTION
**Pull request by FireIsGood**
Refactors config into a new file and then adds the option of `notifyNotFound` (default of `true`) to allow disabling the notification.

- Moved `init.lua` configuration &rarr; `configuration.lua`
- Added `notifyNotFound` to configuration (default: `true`)

Example:

https://github.com/chrisgrieser/nvim-various-textobjs/assets/109556932/0e8fbfd6-af61-4935-b136-3eb545711c24

## Checklist
- [x] Used only camelCase variable names.
- [x] If functionality is added or modified, also made respective changes to the
  README.md (the `.txt` file is auto-generated and does not need to be modified).
- [x] Used conventional commits keywords.

### CODE REVIEW

**Comment by FireIsGood** [comment]
An additional consideration might be to change the individual declarations of `config.lookForwardSmall` and `config.lookForwardBig` to be declared at the top of the file as the line-wise file seems to only use the Big size while the character-wise file seems to only use the Small size.

**Review by chrisgrieser — CHANGES_REQUESTED** [review]
Just an issue with the setup call, otherwise nice work

**Comment by chrisgrieser** [comment]
> An additional consideration might be to change the individual declarations of `config.lookForwardSmall` and `config.lookForwardBig` to be declared at the top of the file as the line-wise file seems to only use the Big size while the character-wise file seems to only use the Small size.

I think there were 1-2 exceptions to this, iirc the url text object.

File: lua/various-textobjs/config.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+---optional setup function
+---@param userConfig? config
+function M.setup(userConfig)
+	M.config = vim.tbl_deep_extend("force", M.config, userConfig or {})
+
+	if M.config.useDefaultKeymaps then
+		require("various-textobjs.default-keymaps").setup(M.config.disabledKeymaps)
+	end
+	return M.config
```
[the remark this answers is not in the document]
Comment (FireIsGood) [reply lua/various-textobjs/config.lua:28]: In the implementation I went with, the `setup()` function both modifies the module's `config` attribute and also returns a copy of it. If I were to use the `setup()` only to set the user config, assigning the config could be done with the `config` attribute instead in the `init.lua` call.

Would this be preferred?

**Review by chrisgrieser — APPROVED** [review]
Looks good to me!

**Comment by chrisgrieser** [comment]
btw, the returning of the config is not really a bad pattern, the return value is just unnecessary because it works have been never used, that is all

**Pull request merged**